include/ffconf.h

```diff
@@ -253,6 +253,15 @@
 /  disk_ioctl() function. */
 
 
+#if defined(CONFIG_FS_FATFS_WIN_ALIGNED)
+#if CONFIG_FS_FATFS_WIN_ALIGNED != 0
+#define FS_FATFS_WIN_ALIGNED   CONFIG_FS_FATFS_WIN_ALIGNED
+#endif /* CONFIG_FS_FATFS_WIN_ALIGNED != 0 */
+#endif /* defined(CONFIG_FS_FATFS_WIN_ALIGNED) */

```

Comment:
Do the `#ifdef` vlock similar way the other Kconfig configured ff parameters are, look for example how ``CONFIG_FS_FATFS_MAX_SS` affects `FF_MAX_SS`.

Reply:
And change the name from `WIN_ALIGNED` to `WINDOW_ALIGNMENT`, because we do not anything to `WIN` we aligned `WIN`  to something.

Reply:
Referring to https://github.com/zephyrproject-rtos/fatfs/pull/9/files#r774606396 if `CONFIG_FS_FATFS_WINDOW_ALIGNMENT` is not defined, default the FS_FATFS_WINDOW_ALIGNMENT to 1

Reply:
Okay, will update the PR again with the above suggestion. 
1) To use follow the #ifdef like what FF_MAX_SS defined
2) change the WIN_ALIGNED to WINDOW_ALIGNMENT
3) default the FS_FATFS_WINDOW_ALIGNMENT to 1